From cf72c26e030b205971627465441b111e7ebaedb4 Mon Sep 17 00:00:00 2001
From: Alex Stewart <alex.stewart@ni.com>
Date: Wed, 12 May 2021 10:48:05 -0500
Subject: [PATCH] nilrt-x86_64.cmake: set legacy toolchain variable

The CMakeLists file now distinguishes between NILRT builds using a
static toolchain archive (legacy toolchain) and builds using a
native-sysroot in an OpenEmbedded build environment.

Set _GRPC_DEVICE_NILRT_LEGACY_TOOLCHAIN from within the legacy toolchain
file, so that build dependencies are set appropriately.

Signed-off-by: Alex Stewart <alex.stewart@ni.com>

Upstream-Status: Pending
---
 scripts/cmake/nilrt-x86_64.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/cmake/nilrt-x86_64.cmake b/scripts/cmake/nilrt-x86_64.cmake
index 7f00969..c007f69 100644
--- a/scripts/cmake/nilrt-x86_64.cmake
+++ b/scripts/cmake/nilrt-x86_64.cmake
@@ -3,6 +3,7 @@
 #----------------------------------------------------------------------
 set(CMAKE_SYSTEM_NAME Linux)
 set(CMAKE_SYSTEM_PROCESSOR x86_64)
+set(_GRPC_DEVICE_NILRT_LEGACY_TOOLCHAIN TRUE)
 
 #----------------------------------------------------------------------
 # Path variables for toolchains
